staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/schemas.go

```diff
@@ -207,6 +207,7 @@ func WithTypeAndObjectMeta(s *schema.Structural) *schema.Structural {
 		return s
 	}
 	result := &schema.Structural{
+		Items:           s.Items,
```

Comment:
Found this in a [`TestZeroValueValidation` integration test failure](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/110135/pull-kubernetes-integration/1548133907776409600).  I'm unclear on why this wasn't observed before, but it's a clear error.

Reply:
Or maybe I'm thinking about this wrong and only object should be allowed at the root?